src/main/java/seedu/address/ui/TempDisplay.java

```diff
@@ -0,0 +1,4 @@
+package seedu.address.ui;
```

Comment:
I believe this file should be deleted, I don't see `TempDisplay` used anywhere else. Could you confirm?